### PR TITLE
LPOT get started update for Devcloud node and mode

### PR DESCRIPTION
**Pull request by bdmoore1**
# Introduction
This template serves 2 purposes and will need to be edited accordingly prior to submitting the PR
- Changes to existing samples
- Submission of new Samples

# Existing Sample Changes
## Description

Made edits to the Running Samples in Devcloud section to enable users to select the correct mode and node to run their sample.

**NOTE**: Prior to accepting pull request, please review the commented sections in the README.md:

1. If your sample runs in batch mode, please delete the section on how to run in interactive mode. Joe Oster requested this because interactive mode can be troublesome so we would rather have users use batch mode when possible.
2. In the node table, if a node is applicable to this sample, please change the node name to **BOLD** so users can easily identify which nodes are available.

Also, I corrected several (R) to show &reg; and the last sentence said 'pratice', I corrected that to 'practice'.

Fixes Issue# 

## External Dependencies

None.

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Implement fixes for ONSAM Jiras
- [ x] Documentation

## How Has This Been Tested?

Text was reviewed and approved by Dan Petre, Donna Boyer, and Joe Oster.

- [ ] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [x ] VSCode
- [ ] When compiling the compliler flag "-Wall -Wformat-security -Werror=format-security" was used

### CODE REVIEW

**Comment by JoeOster** [comment]
@NeoZhangJianyu can you take a look at the description of this PR and the readme that's being altered. I need your input, so I can merge it

**Comment by NeoZhangJianyu** [comment]
@JoeOster 
The word 'tensorflow' in code part should not be replaced by "TensorFlow".
Please rollback from line 100 for 'tensorflow'.

File: AI-and-Analytics/Getting-Started-Samples/LPOT-Sample-for-Tensorflow/README.md [hunk 1]
```diff
@@ -47,20 +47,33 @@ We will learn how to train a CNN model based on Keras with Tensorflow, use LPOT
...
+| What you will learn               | How to use LPOT tool to quantize the AI model based on TensorFlow and speed up the inference on Intel(R) Xeon(R) CPU
 | Time to complete                  | 10 minutes
 
 ## Running Environment
 
-### Running in Devcloud
+### Running Samples In DevCloud (Optional)
 
-If running a sample in the Intel DevCloud, please follow the below steps to build the python environment. Also, remember that you must specify the compute node (CPU) as well as whether to run in batch or interactive mode. For more information, see the [Intel(R) oneAPI AI Analytics Toolkit Get Started Guide](https://devcloud.intel.com/oneapi/get-started/analytics-toolkit/)
+### Run in Interactive Mode
```
Comment (louie-tsai) [inline AI-and-Analytics/Getting-Started-Samples/LPOT-Sample-for-Tensorflow/README.md:57]: Those are general contents for all samples.
Could you move out the contents from specific sample's README and move it to a general place such as README for all AI samples?

ex: https://github.com/oneapi-src/oneAPI-samples/blob/master/AI-and-Analytics/README.md

Comment (bdmoore1) [reply AI-and-Analytics/Getting-Started-Samples/LPOT-Sample-for-Tensorflow/README.md:57]: @louie-tsai  DX (Donna) says that user studies show that users are not looking at the generic readme so we need to have the instructions in the README for each sample. 
I will also add the generic instructions to the AI General README, but I don't want to remove them from the README for each sample.

**Comment by bdmoore1** [comment]
@NeoZhangJianyu now that the tensorflow case is fixed, do you approve this PR?

**Review by JoeOster — APPROVED** [review]

**Pull request merged**